xls/examples/BUILD

```diff
@@ -163,20 +163,12 @@ xls_dslx_library(
 xls_dslx_test(
     name = "fir_filter_dslx_test",
     srcs = ["fir_filter.x"],
-    dslx_test_args = {
-        "compare": "none",
-    },
     deps = [":fir_filter_dslx"],
 )
 
 xls_dslx_test(
     name = "matmul_4x4_dslx_test",
     srcs = ["matmul_4x4.x"],
-    # TODO(rspringer): 2022-05-24: Does not successfully lower to IR.
```

Comment:
Can this TODO be kept to keep a record that we still support lowering this proc to IR?

Reply:
I've change commit to keep this TODO